lib/functions/configuration/interactive.sh

```
@@ -233,14 +233,16 @@ function interactive_config_ask_branch() {
                 legacy)
-                    description="Old stable / Legacy / Vendor kernel"
+                    description="Old stable / Legacy / kernel"
                     ;;
                 edge)
                     description="Bleeding edge / latest possible"
                     ;;
                 cloud)
-                    description="Cloud optimised minimal LTS kernel"
+                    description="Cloud optimized minimal LTS kernel"
                     ;;
+                vendor)
```

Shellcheck (0.10.0)

[error] Couldn't parse this case item. Fix to allow more checks.

(SC1073)